### PR TITLE
tests for improve eula screen feature

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts [hunk 1]
```diff
@@ -9,34 +9,62 @@ const settingsPage = new SettingsPage();
...
-        await t.click(settingsPage.submitConsentsPopupButton);
-        //Verify that I dont have an popup
-        await t.expect(userAgreementPage.userAgreementsPopup.exists).notOk('User Agreements Popup isn\'t shown after accept agreements');
-        //Verify that Welcome page is displayed
-        await t.expect(addRedisDatabasePage.welcomePageTitle.exists).ok('Welcome page is displayed');
-        //Verify I can work with the application
-        await t.click(addRedisDatabasePage.addDatabaseButton);
-        await t.expect(addRedisDatabasePage.addDatabaseManually.exists).ok('User can add a database');
-    });
+test('Verify that user should accept User Agreements to continue working with the application, the Welcome page is opened after user agrees, the encryption enabled by default and specific message', async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts:18]: @ViktarStarastsenka - This isn't e2e, it has  to be covered in JEST as many tests cases

File: tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts [hunk 2]
```diff
@@ -9,34 +9,62 @@ const settingsPage = new SettingsPage();
...
+    await t.expect(userAgreementPage.userAgreementsPopup.exists).ok('User Agreements Popup is shown');
+    //Verify that section with plugin warning is displayed
+    await t.expect(userAgreementPage.pluginSectionWithText.visible).ok('Plugin text is displayed');
+    //Verify that text that is displayed in window is 'While adding new visualization plugins, use files only from trusted authors to avoid automatic execution of malicious code.'
+    const pluginText = userAgreementPage.pluginSectionWithText.innerText;
+    await t.expect(pluginText).eql('To avoid automatic execution of malicious code, when adding new Workbench plugins, use files from trusted authors only.');
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts:24]: Please add error message

File: tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts:18]: It's not my test, but ok, I'll update it

File: tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts (hunk 2 above)
Comment (vlad-dargel) [reply tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts]: fixed

File: tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts]: fixed

[the remark this answers is not in the document]
Comment (vlad-dargel) [reply tests/e2e/tests/critical-path/a-first-start-form/user-agreements-form.e2e.ts]: fixed

File: tests/e2e/pageObjects/user-agreement-page.ts [hunk 3]
```diff
@@ -23,4 +24,8 @@ export class UserAgreementPage {
           await t.click(this.submitButton);
       }
   }
+
+  async getRecommendedSwitcherValue(): Promise<string> {
```
[the remark this answers is not in the document]
Comment (vlad-dargel) [reply tests/e2e/pageObjects/user-agreement-page.ts]: fixed

File: tests/e2e/pageObjects/settings-page.ts [hunk 4]
```diff
@@ -44,7 +45,15 @@ export class SettingsPage {
...
+    async getAnalyticsSwitcherValue(): Promise<string> {
         return await this.switchAnalyticsOption.getAttribute('aria-checked');
     }
+
+    async getNotificationsSwitcherValue(): Promise<string> {
+        return await this.switchNovitifationsOption.getAttribute('aria-checked');
+    }
+
+    async getEulaSwitcherValue(): Promise<string> {
+        return await this.switchEulaOption.getAttribute('aria-checked');
```
[the remark this answers is not in the document]
Comment (vlad-dargel) [reply tests/e2e/pageObjects/settings-page.ts]: fixed

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**